Assets/MixedRealityToolkit/Services/BaseCoreSystem.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.Definitions;
+using Microsoft.MixedReality.Toolkit.Core.Interfaces;
+
+namespace Microsoft.MixedReality.Toolkit.Core.Services
+{ 
+    public abstract class BaseCoreSystem : BaseEventSystem
+    {
+        public BaseCoreSystem(
+            IMixedRealityServiceRegistrar registrar,
+            BaseMixedRealityProfile profile = null) : base()
+        {
+            Registrar = registrar;
+            ConfigurationProfile = profile;
+            Priority = 5; // Core systems have a higher default priority than other services   
```

Comment:
>5 [](start = 23, length = 1)

What's the significance of the number 5? i.e. would it be helpful to have a file with priority constants define what the core system priorities should be, and where the starting point of other service priorities should be?

for example:

public static readonly int CoreServicePriority = 5;
public static readonly int NonCoreServicePriorityStart = 100;

So to manage priorities among non-core you would always do YourPriority + NonCoreServicePriorityStart

Reply:
@wiwei Agreed, I'm not as allergic to magic numbers as some but these priority numbers wig me out. I could see devs arbitrarily giving their service a priority 99 because they don't know the expected range. Probably an issue for a separate PR since it would be a fairly big change.

Reply:
5 was me leaving existing behavior in place. i am definitely open for adding a defined value